crdppf/__init__.py

```diff
@@ -87,14 +92,9 @@ def includeme(config):
     config.add_route('ogcproxy', '/ogcproxy')
     
     # CLIENT VIEWS
-    config.add_view('crdppf.views.entry.Entry', route_name = 'home')
     config.add_view('crdppf.views.entry.Entry', route_name = 'images')
     config.add_view('crdppf.views.entry.Entry', route_name='test')
 
-    # ADMIN VIEWS
```

Comment:
No... if just over your function you define the `@view_config`, then you do not need to define `config.add_view`. These declaration do exactly the same... (they define a view...). This is redundant... and Pyramid 1.6 does not like it... (it crashes...)

For me, the `@view_config` is clearer, thus I would propose that we keep using it...